docs/apps/prowlarr.md

```diff
@@ -22,4 +25,35 @@ sb install prowlarr
 
 ### 3. Setup
 
+***Adding Indexers to Sonarr/Radarr via Prowlarr***
+
+#### Under **Settings >> Apps**
+
+1. Click the big `+` sign to add an App.
+2. Provide a "Name", ie Sonarranime if you have multiple instances.
+![Prowlarr Apps](../images/prowlarr-apps.png)
+3. Provide the url for Prowlarr, this should be `http://prowlarr:9696`
+4. Provide the *local* url for [Sonarr](../apps/sonarr.md)/[Radarr](../apps/radarr.md), ie `http://sonarr:8989` or `http://radarr:7878`
```

Comment:
For consistency and the literratti you should probably follow through with the sonarranime in examples shown

- "2. Provide a "Name", ie Sonarranime if you have multiple instances."
- "4. Provide the *local* url for [Sonarr](../apps/sonarr.md)/[Radarr](../apps/radarr.md), ie `http://sonarr:8989` or `http://radarr:7878`" 

So I would suggest maybe including  `http://sonarranime:8989` in the examples shown in "4"

Reply:
I'll try and get to this tomorrow max! sorry. New class been taking all my time. And the kids starting baseball :p